roles/docker/tasks/main.yml

```diff
@@ -36,6 +36,15 @@
   tags:
     - prebake-for-dev
 
+- stat: path=/var/docker_images
+  register: docker_images
```

Comment:
Yes but this happens at provisioning time, not prebake time. It's 
unpacking the images.

-Erik

On 9 Mar 2016, at 9:23, Madhav Puri wrote:

>> @@ -36,6 +36,15 @@
>> tags:
>>   - prebake-for-dev
>>
>> +- stat: path=/var/docker_images
>> +  register: docker_images
>
> the only function this tag offers is to pre-bake things in the packer 
> image (to lower ansible provision times in vagrant environments) while 
> not special casing them and using same ansible. So it's function is 
> pretty limited. If there is a better way to accomplish this I would be 
> happy to explore that option.
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/ansible/pull/113/files#r55554645